Dockerfile

```diff
@@ -74,6 +74,9 @@ RUN ./kali_fix.sh
 WORKDIR /root/Mobile-Security-Framework-MobSF
 RUN pip3 install -r requirements.txt
 
+#install Postgres dependencies
+RUN pip3 install psycopg2
```

Comment:
Is this required here? 
Should this be moved to `Dockerfile.postgres`?

Reply:
yep agree

Reply:
removed